otel/README.md

```diff
@@ -12,8 +14,7 @@ Host metrics are shown in the default dashboard, but you can send arbitrary metr
 
 Follow the [OpenTelemetry Collector documentation][1] to install the `opentelemetry-collector-contrib` distribution, or any other distribution that includes the Datadog exporter.
 
-The Datadog Agent is **not** needed to export telemetry data to Datadog.
-
+The Datadog Agent is **not** needed to export telemetry data to Datadog in this setup. Check [our public documentation][7] if you want to use the Datadog Agent instead.
```

Comment:
```suggestion
The Datadog Agent is **not** needed to export telemetry data to Datadog in this setup. See [Datadog's OpenTelemetry collector][7] if you want to use the Datadog Agent instead.
```

In general, we avoid using the word "documentation" in links. Instead, we use the title of the link target or a description. I tried to come up with a good description of the target page, but please change it if it is incorrect or misleading.

Reply:
II think the name Product has chosen is "OTLP ingest in the Datadog Agent". What do you think about
```suggestion
The Datadog Agent is **not** needed to export telemetry data to Datadog in this setup. Check [OTLP Ingest in Datadog Agent][7] if you want to use the Datadog Agent instead.
```
instead?